.gitignore

```diff
@@ -4,4 +4,5 @@ base-with-profile/
 rio/target
 observations.edn
 eduhub-validator
-report.html
\ No newline at end of file
+report.html
+target
```

Comment:
ik heb er zelf voor gekozen om ook mijn target e.d. binnen de `rio` directory te houden. Zou dat voor eduxchange ook kunnen?

Reply:
yes, is aangepast